PennMobile/src/main/res/layout/directory_list_item.xml

```diff
@@ -39,9 +39,7 @@
             android:id="@+id/tv_person_name"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:paddingBottom="2dp"
-            android:textSize="20sp"
-            android:textStyle="bold"
+            android:textSize="17sp"
```

Comment:
Can you move these hardcoded values into the styles so that they're easier to change down the line?